presto-product-tests/bin/product-tests-suite-2.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -xeuo pipefail
+
+presto-product-tests/bin/run_on_docker.sh \
+        singlenode -g hdfs_no_impersonation,avro
+
+presto-product-tests/bin/run_on_docker.sh \
+        singlenode-kerberos-hdfs-no-impersonation -g hdfs_no_impersonation
+
+presto-product-tests/bin/run_on_docker.sh \
+        singlenode-hdfs-impersonation -g storage_formats,cli,hdfs_impersonation
+
+presto-product-tests/bin/run_on_docker.sh \
+        singlenode-kerberos-hdfs-impersonation -g storage_formats,cli,hdfs_impersonation,authorization,hive_file_header
```

Comment:
can we wrap this to put each group into separate line?

Reply:
i think the syntax is `-g groups,groups,....`. Any way to specify multiple groups over multiple lines (multiple args)?

Reply:
Maybe wrap the group option onto the next line. This would keep them all aligned:
```shell
presto-product-tests/bin/run_on_docker.sh \
    singlenode-hdfs-impersonation \
        -g storage_formats,cli,hdfs_impersonation

presto-product-tests/bin/run_on_docker.sh \
    singlenode-kerberos-hdfs-impersonation \
        -g storage_formats,cli,hdfs_impersonation,authorization,hive_file_header

Reply:
I wrote a quick test and commons-cli that Tempto is using supports this.

Reply:
It looks like you can repeat the options. I wrote a quick test and commons-cli that Tempto is using supports this.

Reply:
The way Tempto uses the options doesn't allow for repeated options. Let's consider this a follow-up.